.golangci.yml

```diff
@@ -85,8 +85,8 @@ linters-settings:
           - "fmt.Print"
           - "fmt.Printf"
           - "fmt.Println"
-          - "rand.Read"
-          - "sb.WriteString"
+          - "math/rand\\.Read"
+          - "strings\\.Builder\\.WriteString"
```

Comment:
Whats the reason for this / and \\ stuff here ?

Reply:
@mo-c4t

Reply:
this was added by avax. to add some functions to that rule because they are returning errors while linting.  I believe "\\." is the same as "." but / has other meaning, I think the rand function under math lib. and the / is  needed here.